airbyte-db/db-lib/src/test/java/io/airbyte/db/instance/configs/migrations/V0_40_3_001__RemoveActorForeignKeyFromOauthParamsTableTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.instance.configs.migrations;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.airbyte.db.instance.configs.AbstractConfigsDatabaseTest;
+import java.io.IOException;
+import java.sql.SQLException;
+import org.jooq.DSLContext;
+import org.jooq.impl.DSL;
+import org.junit.jupiter.api.Test;
+
+public class V0_40_3_001__RemoveActorForeignKeyFromOauthParamsTableTest extends AbstractConfigsDatabaseTest {
+
+  @Test
+  void test() throws IOException, SQLException {
+    final DSLContext context = getDslContext();
+    V0_32_8_001__AirbyteConfigDatabaseDenormalization.migrate(context);
```

Comment:
You should use the most recent migrations here. Otherwise, there is a risk that other migrations may have changed something that could conflict with this one.

Reply:
I moved to using the `DevDatabaseMigrator` like you had done in the tests for the previous migration (V0_39_17_001__AddStreamDescriptorsToStateTableTest) so that it steps through the previous migrations, but I had to tweak the class a bit so that it allows passing a target version instead of always migrating to whatever the second to last migration version is. I did this because as new migrations are introduced the tests could break (and it does break in my case, since my test just checks that the FK is present before the migration and removed after the migration) [2743a3a](https://github.com/airbytehq/airbyte/pull/16118/commits/2743a3a14ca223a88d13076bacce10202514f2aa)